functions.php

```diff
@@ -368,14 +368,14 @@ function getPasswordPolicy($guid, $connection2)
 
 function getFastFinder($connection2, $guid)
 {
-    $form = Form::create('fastFinder', $_SESSION[$guid]['absoluteURL'].'/indexFindRedirect.php', 'get');
+    $form = Form::create('fastFinder', $gibbon->session->get('absoluteURL').'/indexFindRedirect.php', 'get');
```

Comment:
The `$gibbon` object unfortunately isn't available inside these functions. We plan to eventually refactor and replace the functions in this file, but for now the $_SESSION variables will have to left in place.

Reply:
That's fixed now.